hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
oh, i see. still, i don't like how domain is mixed with API, this is perfect example of tight coupling, especially that calculations will probably be corrected (or parameterized), which would force reafactoring in API